src/components/Progress/README.md

```diff
@@ -1,60 +1,248 @@
-| className       | `String` |          |                           | ClassName of element                                                                                                                     |
-| stackClassName  | `String` |          |                           | ClassName of stack element                                                                                                               |
-
-#### stack
-
-Array of objects:
-
-```js
-{
-    value: number,
-    color?: string,
-    theme?: string,
-    title?: string,
-    className?: string,
-    content?: ReactNode
-}
+<!--GITHUB_BLOCK-->
+
+# Progress
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Progress} from '@gravity-ui/uikit';
+```
+
+`Progress` component indicates current operation progress. It can also be divided into sections.
+
+## Theme
+
+Use `theme` property for specify color of the whole progress or the composite part.
+
+<!--LANDING_BLOCK
+
+<ExampleBlock
+  code={`
+<Progress text="default" value={80} />
+<Progress text="warning" theme="warning" value={70} />
+<Progress text="info" theme="info" value={90} />
+<Progress text="success" theme="success" value={90} />
+<Progress text="danger" theme="danger" value={80} />
+<Progress text="misc" theme="misc" value={60} />
+`}
+>
+  <div style={{width: '30%'}}>
+    <UIKit.Progress text="default" value={80} />
+    <div style={{height: '15px'}} />
+    <UIKit.Progress text="success" theme="success" value={90} />
+  </div>
+  <div style={{width: '30%'}}>
+    <UIKit.Progress text="warning" theme="warning" value={70} />
+    <div style={{height: '15px'}} />
+    <UIKit.Progress text="danger" theme="danger" value={80} />
+  </div>
+  <div style={{width: '30%'}}>
+    <UIKit.Progress text="info" theme="info "value={90} />
+    <div style={{height: '15px'}} />
+    <UIKit.Progress text="misc" theme="misc" value={60} />
+  </div>
+</ExampleBlock>
+
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Progress text="default" value={80} />
+<Progress text="warning" theme="warning" value={70} />
+<Progress text="info" theme="info" value={90} />
+<Progress text="success" theme="success" value={90} />
+<Progress text="danger" theme="danger" value={80} />
+<Progress text="misc" theme="misc" value={60} />
 ```
 
-#### colorStops
+<!--/GITHUB_BLOCK-->
 
-Array of objects:
+## States
 
-```js
-{
-    theme: string,
-    stop?: number = 100
-}
+<!--LANDING_BLOCK
+
+<ExampleBlock
+  code={`
+<Progress text="Loading" theme="misc" value={60} loading={true} />
+`}
+>
+  <div style={{width: '30%'}}>
+    <UIKit.Progress text="Loading" theme="misc" value={60} loading={true} />
+  </div>
+</ExampleBlock>
+
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Progress text="Loading" theme="misc" value={60} loading={true} />
 ```
 
-### Examples
+<!--/GITHUB_BLOCK-->
```

Comment:
What states is this code snippet about? Maybe add some description?

Reply:
There is only Loading
<img width="840" alt="Screenshot 2023-08-25 at 19 53 15" src="https://github.com/gravity-ui/uikit/assets/22904809/88dafdcf-48f0-4d02-af90-555952d4f5cf">

Reply:
Yes, it's clear when you see the preview